src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -141,11 +145,16 @@ void AddEnvironment ()
 		{
 			bool usesMonoAOT = false;
 			bool usesAssemblyPreload = EnablePreloadAssembliesDefault;
+			bool brokenExceptionTransitions = false;
 			uint monoAOTMode = 0;
 			string androidPackageName = null;
 			var environmentVariables = new Dictionary<string, string> (StringComparer.Ordinal);
 			var systemProperties = new Dictionary<string, string> (StringComparer.Ordinal);
 
+			if (!Enum.TryParse (PackageNamingPolicy, out PackageNamingPolicy pnp)) {
+				pnp = PackageNamingPolicyEnum.LowercaseHash;
```

Comment:
As of 88a1d6cd88e68bda208a6eb05d99d802c7800d2e, this should probably default to `PackageNamingPolicy.LowercaseCrc64`.

Reply:
it was a copy-and-paste, oops. I'll change the default in the original location too, then (the default should be really specified somewhere else...)